README.md

```diff
@@ -62,11 +64,10 @@ libraryDependencies += collectionCompat // required for Collection213CrossCompat
 addCompilerPlugin(scalafixSemanticdb)
 scalacOptions ++= List(
   "-Yrangepos",
-  "-Ywarn-unused-import",
-  "-P:semanticdb:synthetics:on"
+  "-P:semanticdb:synthetics:on" // Required by the collection rewrites
 )
 
-scalafixDependencies in ThisBuild += "org.scala-lang.modules" %% "scala-collection-migrations" % Dependencies.collectionCompatVersion
+scalafixDependencies in ThisBuild += "org.scala-lang.modules" %% "scala-collection-migrations" % collectionCompatVersion
```

Comment:
nitpick, is the variable for the version necessary? I think it's more copy-paste friendly if the module id is self-contained